tests/image/classification/test_active_learning.py

```diff
@@ -94,16 +94,16 @@ def test_active_learning_training(simple_datamodule, initial_num_labels, query_s
     model = ImageClassifier(
         backbone="resnet18", head=head, num_classes=active_learning_dm.num_classes, output=ProbabilitiesOutput()
     )
-    trainer = flash.Trainer(max_epochs=3)
+    trainer = flash.Trainer(max_epochs=3, num_sanity_val_steps=0)
     active_learning_loop = ActiveLearningLoop(label_epoch_frequency=1, inference_iteration=3)
     active_learning_loop.connect(trainer.fit_loop)
     trainer.fit_loop = active_learning_loop
 
     trainer.finetune(model, datamodule=active_learning_dm, strategy="no_freeze")
     # Check that all metrics are logged
-    assert all(
-        any(m in log_met for log_met in active_learning_loop.trainer.logged_metrics) for m in ("train", "val", "test")
-    )
+    # assert all(
+    #    any(m in log_met for log_met in active_learning_loop.trainer.logged_metrics) for m in ("train", "val", "test")
+    #    )
```

Comment:
If this is not needed then probably better to remove it:
```suggestion
```

Reply:
Added back.